Adds Cash App case study (#173)
lightningdevkit/lightningdevkit.org: docs/_blog/cashapp-enables-lightning-withdrawals-and-deposits-using-ldk.md

---

> title: "CashApp enables lightning withdrawals & deposits using LDK"

Reviewer: ```suggestion
title: "Cash App Enables Lightning Withdrawals & Deposits Using LDK"
```

> description: "Learn how the CashApp team integrated with the Lightning Network, allowing its users to send and receive bitcoin instantly."

Reviewer: ```suggestion
description: "Learn how Cash App integrated with the Lightning Network, allowing its users to send and receive bitcoin instantly."
```

date: "2022-11-01"
authors:
  - Ryan Loomba
tags:
  - Case Studies 
--- 

# Overview

> Our project is the Cash App lightning wallet. We power all lightning bitcoin withdrawals and deposits within Cash App.

Reviewer: This first sentence sounds funny. I think you can just leave out "our project", but need to think about it.

> Cash App is a the #1 finance app in the App Store – we have extremely high standards for product development and the infrastructure that powers all our cool features. We were looking for a solution that would ensure high reliability, high scalability, and a stellar developer experience. Cash App is the financial hub for millions of people and often the first place an individual acquires bitcoin, so having a slick user experience, and the infrastructure to onboard lots of new users is extremely important. As a publicly traded company, we take security very seriously, so having the ability to customize our wallet implementation to adhere to our security standards was necessary.

Reviewer: ```suggestion
Cash App is the #1 finance app in the App Store – we have extremely high standards for product development and the infrastructure that powers all our cool features. We were looking for a solution that would ensure high reliability, high scalability, and a stellar developer experience. Cash App is the financial hub for millions of people and often the first place an individual acquires bitcoin, so having a slick user experience, and the infrastructure to onboard lots of new users is extremely important. As a publicly traded company, we take security very seriously, so having the ability to customize our wallet implementation to adhere to our security standards was necessary.
```
